src/csvcubed/readers/skoscodelistreader.py

```diff
@@ -27,29 +30,16 @@ def extract_code_list_concept_scheme_info(
       `concept_uri_template` uses the standard `notation` uri template variable even if the underlying file uses a
        different column name.
     """
-    table_schema_result = get_first_table_schema(code_list_csvw_path)
-    if table_schema_result is None:
-        raise ValueError(f"Unable to find tableSchema in {code_list_csvw_path}")
+    table_schema_manager = TableSchemaManager(code_list_csvw_path)
+    rdf_graph = table_schema_manager.load_json_ld_to_rdflib_graph()
```

Comment:
Do you think we should move this function into the `__post_init__` so the caller has less to do?

Reply:
Maybe the failed to load exception too?

Reply:
Good shout. I have updated accordingly.